tests/foreman/ui/test_contentaccess.py

```diff
@@ -0,0 +1,242 @@
+
+        :steps:
+
+            1. Access Content-Host Packages tab.
+
+        :CaseAutomation: notautomated
+
+        :expectedresults:
+            1. All packages are available independent
+               of subscription because Golden Ticket is enabled.
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @tier1
+    @stubbed()
+    def test_positive_visual_indicator_on_hosts_subscription(self):
+        """Access content hosts subscription tab and assert a visual indicator
+        is present highlighting that organization hosts have unrestricted
+        access to repository content.
+
+        :id: f8fc0bd2-c92f-4706-9921-4e331762170d
+
+        :steps:
+
+            1. Access Content-Host Subscription tab.
+
+        :CaseAutomation: notautomated
+
+        :expectedresults:
+            1. A visual alert is present at the top of the subscription tab
+               saying: "Access to repositories is unrestricted in
+               this organizataion. Hosts can consume all repositories available
+               in the Content View they are registered to, regardless of
+               subscription status".
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @tier1
+    @stubbed()
+    def test_positive_visual_indicator_on_activation_key_details(self):
+        """Access AK details subscription tab and assert a visual indicator
+        is present highlighting that organization hosts have unrestricted
+        access to repository content.
+
+        :id: 94ba1113-11cb-43b2-882e-bf45b5355d9b
+
+        :steps:
+
+            1. Access Ak details Subscription tab.
+
+        :CaseAutomation: notautomated
+
+        :expectedresults:
+            1. A visual alert is present at the top of the subscription tab
+               saying: "Access to repositories is unrestricted in this
+               organizataion. Hosts can consume all repositories available in
+               the Content View they are registered to, regardless of
+               subscription status".
+
+        :CaseImportance: Critical
+        """
+
+    @run_only_on('sat')
+    @tier1
+    @stubbed()
+    def test_positive_visual_indicator_on_manifest(self):
+        """Access org manifest page and assert a visual indicator
+        is present highlighting that organization hosts have unrestricted
+        access to repository content.
+
+        :id: a9c2d2b7-17ab-441b-978d-24dc80f35a4b
+
+        :steps:
+
+            1. Access org manifest page.
+
+        :CaseAutomation: notautomated
+
+        :expectedresults:
+            1. A visual alert is present at the top of the
+               subscription tab saying: "Access to repositories is unrestricted
+               in this organizataion. Hosts can consume all repositories
+               available in the Content View they are registered to, regardless
+               of subscription status".
+
+        :CaseImportance: Critical
```

Comment:
@rplevka I just used the template from other stubs, which `:CaseImportance:` do you think I should use here?